ocha_snap.module

```diff
@@ -41,14 +41,14 @@ function ocha_snap_credentials() {
  * PDF generator.
  *
  * @param string $url
- *   The URL to generate the PDF for.
+ *   The URL to generate the PDF /PNGfor.
```

Comment:
missing space

Reply:
I have added/reverted the changes as suggested.